project/threads/templates/threads/static_templates/landing.html

```diff
@@ -43,12 +44,9 @@
                 <div class="row">
                     <div class="white cushion">
                         <div class="container section">
-                            <h3 class="content-1-title bold">CiviWiki is a non-profit community working to develop a democratic engagement web system.</h3>
+                            <h3 class="content-1-title bold">{% trans 'CiviWiki is a non-profit community working to develop a democratic engagement web system' %} .</h3>
                             <div class="accent-line"></div>
-                            <p class="content-1-text section">
-                                At CiviWiki we believe democracy works best when citizens are educated on issues and participate in the democratic process.
-                                <br>
-                                <br> We are a non-profit, non-partisan, community; working to build an open-source platform where citizens discuss the problems they face, discover solutions, and petition their representative to enact change.
+                            <p class="content-1-text section"> {% trans 'At CiviWiki we believe democracy works best when citizens are educated on issues and participate in the democratic process.<br><br> We are a non-profit, non-partisan, community; working to build an open-source platform where citizens discuss the problems they face, discover solutions, and petition their representative to enact change.' %}
```

Comment:
Let's break this long text into two paragraphs, rather than use `br` tags.
```suggestion
                            <p class="content-1-text section"> {% trans 'At CiviWiki we believe democracy works best when citizens are educated on issues and participate in the democratic process.' %}
                            <p class="content-1-text section">We are a non-profit, non-partisan, community; working to build an open-source platform where citizens discuss the problems they face, discover solutions, and enact change.</p>
```